### PR TITLE
[RF] Use niworkflows upstreamed interfaces and utilities

### PR DESCRIPTION
**Pull request by oesteban**
Left reports generation for another PR.

Updating with SignalExtraction as well (poldracklab/niworkflows#265)

### CODE REVIEW

File: fmriprep/__about__.py [hunk 1]
```diff
@@ -85,53 +86,54 @@
...
-    'indexed_gzip>=0.8.8',
+    'pyyaml',
     'scikit-image',
+    'scikit-learn',
+    'seaborn',
+    'sklearn',
+    'statsmodels',
+    'tedana>=0.0.5',
     'versioneer',
-    'pyyaml',
```
[the remark this answers is not in the document]
Comment (oesteban) [reply fmriprep/__about__.py]: I think `future` is there for the wrapper. Other than that, I'll check others and clear up.

[the remark this answers is not in the document]
Comment (oesteban) [reply fmriprep/__about__.py]: (although, I'm checking and I don't see any `__future__` anywhere)

[the remark this answers is not in the document]
Comment (effigies) [reply fmriprep/__about__.py]: The wrapper has its own `setup.py`.

[the remark this answers is not in the document]
Comment (oesteban) [reply fmriprep/__about__.py]: I've removed all dependencies in your list. None had an actual import.

[the remark this answers is not in the document]
Comment (effigies) [reply fmriprep/__about__.py]: Ah, we also don't use seaborn or versioneer. (Versioneer is just the tool for creating versioning infrastructure; it doesn't depend on an installed versioneer to work.)

File: fmriprep/__about__.py [hunk 2]
```diff
@@ -85,53 +86,49 @@
...
-    'nitime',
-    'nipype>=1.1.6',
-    'niworkflows==0.5.1',
-    'tedana>=0.0.5',
-    'statsmodels',
-    'seaborn',
-    'indexed_gzip>=0.8.8',
+    'pyyaml',
     'scikit-image',
+    'seaborn',
```
Comment (effigies) [inline fmriprep/__about__.py]:
```suggestion
```

File: fmriprep/__about__.py [hunk 3]
```diff
@@ -85,53 +86,49 @@
...
+    'statsmodels',
+    'tedana>=0.0.5',
     'versioneer',
```
Comment (effigies) [inline fmriprep/__about__.py]:
```suggestion
```

File: fmriprep/__about__.py [hunk 4]
```diff
@@ -4,42 +4,43 @@
...
 from ._version import get_versions
 __version__ = get_versions()['version']
 del get_versions
 
 __author__ = 'The CRN developers'
-__copyright__ = 'Copyright 2018, Center for Reproducible Neuroscience, Stanford University'
-__credits__ = ['Craig Moodie', 'Ross Blair', 'Oscar Esteban', 'Chris Gorgolewski',
-               'Shoshana Berleant', 'Christopher J. Markiewicz', 'Russell A. Poldrack']
+__copyright__ = 'Copyright {}, Center for Reproducible Neuroscience, Stanford University'.format(
+    datetime.now().year)
```
[the remark this answers is not in the document]
Comment (oesteban) [reply fmriprep/__about__.py]: maybe `2016-{}`?. You mean that the copyright should define the starting date?

File: REFERENCES.md [hunk 5]
```diff
@@ -35,7 +35,6 @@
 | nipype | https://doi.org/10.3389/fninf.2011.00013 https://doi.org/10.5281/zenodo.581704 | https://github.com/nipy/nipype/ |
 | convert3d | | https://sourceforge.net/projects/c3d/ |
 | **Graphics**
-| seaborn | https://doi.org/10.5281/zenodo.883859 | https://github.com/mwaskom/seaborn |
```
Comment (effigies) [inline REFERENCES.md]: We do still use it, just not as a direct import. https://github.com/poldracklab/niworkflows/search?utf8=%E2%9C%93&q=seaborn&type=

Comment (oesteban) [reply REFERENCES.md]: I'll update the palettes in niworkflows and credit seaborn over there. Would that be okay?

File: fmriprep/__about__.py (hunk 4 above)
[the remark this answers is not in the document]
Comment (effigies) [reply fmriprep/__about__.py]: Just that if we release tomorrow, and then in 20 years, somebody does:

```Python
>>> import fmriprep
>>> fmriprep.__version__
1.3.0
>>> fmriprep.__copyright__
'Copyright 2038, ...'
```

I think updating this once a year as long as it's actively edited isn't such a burden.

File: REFERENCES.md (hunk 5 above)
Comment (effigies) [reply REFERENCES.md]: I'd rather do it on the package that people actually use.

File: fmriprep/__about__.py (hunk 4 above)
[the remark this answers is not in the document]
Comment (oesteban) [reply fmriprep/__about__.py]: Gotcha

File: REFERENCES.md (hunk 5 above)
Comment (oesteban) [reply REFERENCES.md]: Okay, rolled back.

**Comment by oesteban** [comment]
Still need to investigate why ds054 does not generate the errored report anymore and why masks don't show green contours at all (regression tests)

**Comment by oesteban** [comment]
@effigies I'll fix the indentation problem

ds054 issue is now resolved.

Okay, I run the test_pytest locally and the masks being generated are exactly the same as the originals, and for that reason the green contour is invisible. To double-check, I opened one report with inkscape and removed layers until I got to the green contour. They are there, right below the red contours.

So, in principle, this PR is ready to be merged. Thumbs-up?

**Comment by effigies** [comment]
I'll try to have a look through tonight.

File: docs/workflows.rst [hunk 6]
```diff
@@ -79,7 +79,7 @@ T1w/T2w preprocessing
                               num_t1w=1)
 
 The anatomical sub-workflow begins by constructing an average image by
-:ref:`conforming <conformation>` all found T1w images to RAS orientation and
+conforming all found T1w images to RAS orientation and
```
Comment (oesteban) [inline docs/workflows.rst]: this will be back on when we start generating documentation for smriprep.

**Comment by oesteban** [comment]
Good to go?

**Comment by effigies** [comment]
Oh, sorry. Let me have a look through right now.

**Pull request merged**